WordPress/Sniffs/DB/PreparedSQLPlaceholdersSniff.php

```diff
@@ -43,6 +43,77 @@
 class PreparedSQLPlaceholdersSniff extends Sniff {
 
 	/**
+	 * These regexes copied from http://php.net/manual/en/function.sprintf.php#93552
+	 * and adjusted for limitations in `$wpdb->prepare()`.
+	 *
+	 * Near duplicate of the one used in the WP.I18n sniff, but with less types allowed.
+	 *
+	 * Note: The regex delimiters and modifiers are not included to allow this regex to be
+	 * concatenated together with other regex partials.
+	 *
+	 * @since 0.14.0
+	 *
+	 * @var string
+	 */
+	const PREPARE_PLACEHOLDER_REGEX = '(?:
+		(?<![^%]%)                     # Don\'t match a literal % (%%), including when it could overlap with a placeholder.
+		(?:
+			%                          # Start of placeholder.
+			(?:[0-9]+\\\\?\$)?         # Optional ordering of the placeholders.
+			[+-]?                      # Optional sign specifier.
+			(?:
+				(?:0|\'.)?                 # Optional padding specifier - excluding the space.
+				-?                         # Optional alignment specifier.
+				[0-9]*                     # Optional width specifier.
+				(?:\.(?:[ 0]|\'.)?[0-9]+)? # Optional precision specifier with optional padding character.
+				|                      # Only recognize the space as padding in combination with a width specifier.
```

Comment:
Alignment of comment seems to off by 4.

Reply:
That's on purpose to make it clear that it divides the regex part this is in ( ... | ... ) and the docs explain what the second part is about.